x-pack/plugins/profiling/server/routes/setup.ts

```diff
@@ -99,14 +122,18 @@ export function registerSetupRoute({
           });
         }
 
-        const verifyFunctions = [
-          validateCollectorPackagePolicy,
+        /**
+         * Performe advanced verification in case the first step failed.
+         */
+        const adminUsersVerifyFunctions = [
           validateMaximumBuckets,
           validateResourceManagement,
           validateSecurityRole,
-          validateSymbolizerPackagePolicy,
         ];
-        const partialStates = await Promise.all(verifyFunctions.map((fn) => fn(setupOptions)));
+
+        const partialStates = await Promise.all(
+          adminUsersVerifyFunctions.map((fn) => fn(setupOptions))
+        );
         const mergedState = mergePartialSetupStates(state, partialStates);
```

Comment:
```suggestion
        const mergedStateForAdmin = mergePartialSetupStates(mergedStateForViewer, partialStates);
```

By merging with `mergedStateForViewer` as the base value instead, I think it makes it clearer to me that as we proceed through the function that we are successively filling out the `SetupState` structure.